src/Psy/Configuration.php

```diff
@@ -115,6 +118,10 @@ public function init()
         if ($configFile = $this->getConfigFile()) {
             $this->loadConfigFile($configFile);
         }
+
+        if (!$this->configFile && $localConfig = $this->getLocalConfigFile()) {
```

Comment:
I'd like to see a test case to assert this, just to keep us from making a change that regresses in the future, once we don't have the context of this pull request :)